generators/client/templates/angular/src/main/webapp/app/account/activate/_activate.component.ts

```diff
@@ -16,18 +17,27 @@ export class ActivateComponent implements OnInit {
     modalRef: NgbModalRef;
 
     constructor(
+        <%_ if (enableTranslation) { _%>
+        private jhiLanguageService: JhiLanguageService,
+        <%_ } _%>
         private activate: Activate,
         private loginModalService: LoginModalService,
-        private trans: Transition
-    ) {}
+        private route: ActivatedRoute
+    ) {
+        <%_ if (enableTranslation) { _%>
+        this.jhiLanguageService.setLocations(['activate']);
```

Comment:
this also can be stripped via same regex. remove unwanted regex for the ui router part. let me know if you need help with that